src/snowflake/snowpark/functions.py

```diff
@@ -5390,6 +5390,60 @@ def object_pick(obj: ColumnOrName, key1: ColumnOrName, *keys: ColumnOrName) -> C
     return builtin("object_pick")(o, k1, *ks)
 
 
+def vector_cosine_distance(v1: ColumnOrName, v2: ColumnOrName) -> Column:
+    """Returns the cosine distance between two vectors.
```

Comment:
Can you please add `"of equal dimension and element type"` to each of these return clauses?